vars/folioHelm.groovy

```diff
@@ -171,6 +171,13 @@ String generateModuleValues(RancherNamespace ns, String moduleName, String modul
         moduleConfig << [readWriteSplitEnabled: "true"]
     }
 
+    //consortium_enabled env.variable
+    if (Constants.CONSORTIUM_ENABLED.contains(moduleName) && ns.addConsortiumEvnVariable){
+        moduleConfig << [CONSORTIUM_ENABLED: "true"]
```

Comment:
This will not work as it will add this variable to the root level of values.yaml and not to the extraEnv: |